packages/cfa-template/README.md

```diff
@@ -1,10 +1,45 @@
 # cfa-template
 
-This is the official base template for [Create Frigg App](https://github.com/friggframework/create-frigg-app).
+This is the official base template for [Create Frigg App](https://github.com/friggframework/create-frigg-app), a CLI tool for generating new Frigg applications.
 
-If you don't specify a template (for example, `--template typescript`), this template will be used by default.
+The primary purpose of this template is to provide a structured starting point for developing applications with the Frigg framework.
```

Comment:
We _are_ going to be adding additional templates, with this one being the default